src/Stratis.Bitcoin.Features.SignalR/Broadcasters/BroadcasterBase.cs

```diff
@@ -0,0 +1,57 @@
+namespace Stratis.Bitcoin.Features.SignalR.Broadcasters
+{
+    using System;
```

Comment:
`using`s should be at the top of the file.

Reply:
Done